core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1073,6 +1081,9 @@ class DAGScheduler(
           handleExecutorLost(bmAddress.executorId, fetchFailed = true, Some(task.epoch))
         }
 
+      case TaskCommitDenied(jobID, splitID, attemptID) =>
+        // Do nothing here, left up to the TaskScheduler to decide how to handle denied commits
```

Comment:
I think that we actually need to handle this in TaskScheduler, too, to make sure that commit denials don't count towards the task failure count.